ios/Makefile

```diff
@@ -22,9 +31,42 @@ archive:
 		-scheme $(scheme_name) \
 		-archivePath $(archive_path)
 
-upload:
+export_ipa:
 	xcodebuild \
 		-exportArchive \
 		-archivePath $(archive_path) \
+		-exportPath ./build \
 		-exportOptionsPlist exportOptions.plist \
 		-allowProvisioningUpdates
+
+upload_ipa:
+	# Validte the app before uploading, to catch any possible issues.
+	xcrun altool --validate-app \
```

Comment:
We have to use `altool` as uploading with `xcodebuild` by adding App Store Connect authorisation is available only with Xcode 13. We could migrate to using xcodebuild once GitHub will support building with Xcode 13.

Reply:
You're in luck, [happening next week](https://github.com/actions/virtual-environments/issues/4800), although it's already available in [version 11](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#xcode)

Reply:
Switched to make use of Xcode 13, as it is already available. At the same time I have kept the usage of altool, since uploading through xcodebuild seems to not be very reliable in CI environment.